Source/WebKit/UIProcess/API/glib/WebKitSettings.cpp

```diff
@@ -172,6 +172,7 @@ enum {
     PROP_ENABLE_MEDIA,
     PROP_MEDIA_CONTENT_TYPES_REQUIRING_HARDWARE_SUPPORT,
     PROP_ENABLE_WEBRTC,
+    PROP_ENABLE_WEBSECURITY,
```

Comment:
WEB_SECURITY? same for the property name and getter/setter

Reply:
I know all other settings are `enable-foo`, but in this case I think I would use `disable-web-security` to make even more explicit that this setting is to disable security.